src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointAlignment.java

```diff
@@ -0,0 +1,174 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Holds information about a split alignment of a contig, which may represent an SV breakpoint. Each BreakpointAlignment
+ * represents the junction on the contig of two aligned regions. For example, if a contig aligns to three different regions
+ * of the genome (with one primary and two supplementary alignment records), there will be two BreakpointAlignment
+ * objects created, one to represent each junction between alignment regions:
+ *
+ * Example Contig:
+ * ACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTGCACTGACTG
+ * Alignment regions:
+ * |---------1:100-200------------|
+ *                                 |----------2:100-200------------------|
+ *                                                                       |----------3:100-200-----------------|
+ * Assmbled breakpoints:
+ * 1) links 1:100-200 to 2:100-200
+ * 2) links 2:100-200 to 3:100-200
+ *
+ * Inserted sequence contains portions of the contig that are aligned to neither region, and therefore may be inserted in
+ * the sample. For example, a translocation breakpoint with a microinsertion:
+ *
+ * Contig:
+ * ACTGACTGACTGACTGACTGACTGACTGACTGACTGACTGACTG
+ * Alignment regions:
+ * |-----1:100-200-------|
+ *                          |----2:100-200-----|
+ * Inserted sequence:
+ *  GA
+ *
+ * Homology represents ambiguity about the exact location of the breakpoint. For example, in this case one alignment
+ * region ends with "AC" and the next begins with AC, so we don't know if the AC truly belongs with the first or
+ * second alignment region.
+ *
+ * Contig:
+ * ACTGACTGACTGACTGACTGACTGACTGACTGACTGACTGACTG
+ * Alignment regions:
+ * |-----1:100-200-------|
+ *                    |-----2:100-200----------|
+ * Homology:
+ *  AC
+ */
+class BreakpointAlignment {
+    public static final String NO_SEQUENCE = "none";
+    String contigId;
+    AlignmentRegion region1;
+    AlignmentRegion region2;
+    String insertedSequence;
+    String homology;
+    List<String> insertionMappings;
+
+    public BreakpointAlignment(final String contigId, final AlignmentRegion region1, final AlignmentRegion region2, final String insertedSequence, final String homology, final List<String> insertionMappings) {
```

Comment:
Would be better if this documentation is added to clear confusion:
`Assumes {@code region1} has a lower {@link AlignmentRegion#startInAssembledContig} than {@code region2}.`